example/nomad/tempo-distributed/tempo.hcl

```diff
@@ -0,0 +1,546 @@
+variable "version" {
+  type        = string
+  description = "Tempo version"
+  default     = "2.3.1"
+}
+
+variable "s3_access_key_id" {
+  type        = string
+  description = "S3 Access Key ID"
+  default     = "any"
+}
+
+variable "s3_secret_access_key" {
+  type        = string
+  description = "S3 Secret Access Key ID"
+  default     = "any"
+}
+
+job "tempo" {
+  datacenters = ["*"]
+
+  group "metrics-generator" {
+    count = 1
+
+    network {
+      port "http" {}
+      port "grpc" {}
+    }
+
+    service {
+      name = "tempo-metrics-generator"
+      port = "http"
+      tags = [
+        "traefik.enable=true",
+        "traefik.http.routers.tempo-metrics-generator.rule=Host(`tempo-metrics-generator.service.consul`)"
+      ]
+      check {
+        name     = "metrics-generator"
+        port     = "http"
+        type     = "http"
+        path     = "/ready"
+        interval = "20s"
+        timeout  = "1s"
+      }
+    }
+
+    service {
+      name = "tempo-metrics-generator-grpc"
+      port = "grpc"
+      tags = [
+        "grpc",
+        "traefik.enable=true",
+        "traefik.http.routers.tempo-metrics-generator-grpc.rule=Host(`tempo-metrics-generator-grpc.service.consul`)",

```

Comment:
what do these traefik tags do? is this setting up a service mesh? is that standard in nomad?

Reply:
You are correct on this. There is no need for traefik tags. I have used traefik tags in my env. I will remove the tags.